### ISSUE TITLE
Customized Scrollbars?

### ISSUE THREAD

**Issue by apersonnamedmike** [muckSponge/MaterialFox#62]
Really enjoying using this theme! I was wondering if it's possible for scrollbars to be themed as well? left is android messages in chrome, and the right is current firefox with materialfox from a few days ago.

![image](https://user-images.githubusercontent.com/10731372/51865022-8f253d80-2302-11e9-8ce5-8ebcdc416606.png) ![image](https://user-images.githubusercontent.com/10731372/51865057-a3693a80-2302-11e9-8803-2264fc49bf3e.png)

**Comment by scorthyn** [comment]
THAT would be awesome. Im currently using these overlay scrollbars and its awesome but would be cool if they were implemented in the skin. https://github.com/endeavoursc/firefox-overlay-scrollbars-win10

**Comment by apersonnamedmike** [comment]
Oh nice! I can use that overlay in the mean time, but I agree that it'd be really nice to have it in the skin itself.

**Comment by scorthyn** [comment]
yeah just copy the code in userChrome.css scrollbar to your already userchrome.css and you will be able to have the material skin with that overlay

**Comment by Gigobitto** [comment]
I would love to at the very least have the choice to decide; why not add it but let the single user decide?

**Comment by redoxeon** [comment]
I'm not sure if you want to close this issue, but it seems like in at least a few cases that Firefox has started allowing having styled scrollbars. My particular example of messages.google.com/web has small and colored scrollbars now that closely resemble the ones found in Chrome now, and some other websites have styled scrollbars as well.
